### DISCUSSION TITLE
Connect 2 node from 2 machine

### DISCUSSION THREAD

**Discussion by c1kk** [ravendb/ravendb#13370] [Q&A]
I want to create a cluster have 1 single Node (Leader) and then i want to add another node from another PC to my cluster. And i dont know why it error. Thank you
![Error](https://user-images.githubusercontent.com/92294829/147510772-aa659634-b748-4395-8884-c987864526b8.PNG)
![Create a first Node](https://user-images.githubusercontent.com/92294829/147510774-4ecbc8b6-7233-4c78-9a7d-a7037dc5324e.PNG)
.

**Comment by ayende** [comment]
You can see a video walking you through the process here: https://youtu.be/K-2iZ_lJVag

Basically, you setup _two_ nodes during the setup process and then apply the result from the first setup wizard to the second.

The error you got is about DNS refresh times, you may want to point your DNS to `8.8.8.8` directly, or just wait for your DNS to refresh.

**Reply by c1kk** [reply]
About IP address, i dont know what address need to use for my issue. What IP,HTTP,TCP node A should use and also for node B.

**Reply by ayende** [reply]
Where are you going to run those nodes?

**Reply by c1kk** [reply]
on my PC

**Reply by c1kk** [reply]
run window10

**Reply by c1kk** [reply]
node A i will run on my PC and node B i will on my laptop

**Reply by ayende** [reply]
Then you'll need to get the ips of both of those (probably on the local network).
You can get that via `ipconfig`

**Reply by c1kk** [reply]
And what i need to do next ??

**Reply by ayende** [reply]
Then you follow the video and create a two node cluster with the provided IPs.
Make sure that the ports (443 and 38888) are open in the firewall.

**Reply by c1kk** [reply]
what HTTPS and TCP port i need to use ?

**Reply by ayende** [reply]
TCP is usually `38888`, HTTPS is typically `443`.
You are free to chosen others, of course.
